src/reanimated2/hook/useAnimatedStyle.ts

```diff
@@ -37,7 +37,7 @@ interface AnimatedState {
   isAnimationCancelled: boolean;
 }
 
-interface AnimationRef {
+interface PropUpdaterData {
```

Comment:
We already use the term "prop updater" for `JSPropsUpdater`, but this thing looks way different, we'll need to come up with a different name.

Reply:
Name for this thing is very hard 😭